packages/terra-button-group/tests/wdio/button-group-spec.js

```diff
@@ -1,109 +1,153 @@
   });
 
-  describe('Large', () => {
-    beforeEach(() => browser.url('/#/tests/button-group-tests/large-button-group'));
+  describe('Disabled Buttons', () => {
+    beforeEach(() => browser.url('/#/tests/button-group-tests/not-selectable-button-group'));
     Terra.should.beAccessible();
     Terra.should.matchScreenshot();
   });
 
-  describe('Huge', () => {
-    beforeEach(() => browser.url('/#/tests/button-group-tests/huge-button-group'));
-    Terra.should.beAccessible();
-    Terra.should.matchScreenshot();
-  });
+  describe('onChange', () => {
+    before(() => browser.url('/#/tests/button-group-tests/on-change-button-group'));
+    it('should select the first button', () => {
+      browser.keys('Tab');
+      browser.keys('Space');
+      expect(browser.getText('#selected-key')).to.equal('1');
+    });
+
+    Terra.should.themeEachCustomProperty({
+      '--terra-button-group-button-active-background-color': '#8bc2f9',
+      '--terra-button-group-button-active-and-hover-background-color': '#8bc2f9',
+      '--terra-button-group-button-active-and-focus-background-color': '#8bc2f9',
+    });
 
-  describe('Compact', () => {
-    beforeEach(() => browser.url('/#/tests/button-group-tests/compact-button-group'));
     Terra.should.beAccessible();
-    Terra.should.matchScreenshot();
-  });
+    Terra.should.matchScreenshot('button1-selected');
+
+    it('should select the second button', () => {
+      browser.keys('Tab');
+      browser.keys('Space');
+      expect(browser.getText('#selected-key')).to.equal('2');
+    });
 
-  describe('Icon Default', () => {
-    beforeEach(() => browser.url('/#/tests/button-group-tests/icon-default-button-group'));
     Terra.should.beAccessible();
-    Terra.should.matchScreenshot();
-  });
+    Terra.should.matchScreenshot('button2-selected');
+
+    it('should select the fourth button', () => {
+      browser.keys('Tab');
+      browser.keys('Tab');
+      browser.keys('Space');
+      expect(browser.getText('#selected-key')).to.equal('4');
+    });
 
-  describe('Icon Reversed', () => {
-    beforeEach(() => browser.url('/#/tests/button-group-tests/icon-reversed-button-group'));
     Terra.should.beAccessible();
-    Terra.should.matchScreenshot();
+    Terra.should.matchScreenshot('button4-selected');
   });
 
-  describe('Icon Only', () => {
-    beforeEach(() => browser.url('/#/tests/button-group-tests/icon-only-button-group'));
-    // TODO: remove button-name exception once https://github.com/cerner/terra-core/issues/1058 is closed
-    Terra.should.beAccessible({
-      rules: { 'button-name': { enabled: false } },
+  describe('Single-Select', () => {
+    before(() => browser.url('/#/tests/button-group-tests/single-select-button-group'));
+    it('should select the first button', () => {
+      browser.keys('Tab');
+      browser.keys('Space');
+      expect(browser.getText('#selected-key')).to.equal('1');
+    });
+
+    Terra.should.themeEachCustomProperty({
```

Comment:
You only need to test these themeable variables once, it's not necessary to do it in each of the selectable test pages.

Reply:
I'm doing this to give it a lighter background color so that the test doesn't complain about insufficient color contrast.

Reply:
That's pretty buggy that the theme tests will effect the accessibility tests. If any of default theme values fail accessibility then we need to get different values from @neilpfeiffer

Reply:
That makes sense.

@neilpfeiffer  According to the open-source theme, below are the styles for a selected button in a button group. The contrast for these colors are insufficient for accessibility. Should these be updated to have a larger contrast?
`background-color` :  `#b2b2b2`
`color` : `#434a4d`

For the Button component, its `background-color` and `color` for a neutral variant when selected are `rgba(0, 0, 0, 0.3)`  and `#434a4d` respectively and the contrast is also  insufficient. This may also need to up updated as well.